Domain/Sia.Core/Authentication/KeyVault/AzureSecretVault.cs

```diff
@@ -25,10 +23,12 @@ public AzureSecretVault(KeyVaultConfiguration configuration)
         {
             try
             {
-                var secret = await GetKeyVaultClient().GetSecretAsync(_config.Vault + _secretsEndpoint + secretName).ConfigureAwait(false);
+                var secret = await GetKeyVaultClient()
+                    .GetSecretAsync(_config.Vault + _secretsEndpoint + secretName)
+                    .ConfigureAwait(continueOnCapturedContext: false);
                 return secret.Value;
             }
-            catch (KeyVaultErrorException ex)
+            catch (KeyVaultErrorException)
```

Comment:
The same thing came across my mind as I was deleting the `ex`s. 

To that end I'll see if I can add logging into these or just revert the change so that the warnings stay.

Reply:
Instead of passing a new logger into each instance of a class, this post recommends a logger in each instance that is instantiated from a static factory which is configured at startup. Logging scopes are defined within `using` blocks: https://msdn.microsoft.com/en-us/magazine/mt694089.aspx

This would require a larger effort than is in scope here. I've opened a work item for it.